config/config.msft.yaml

```diff
@@ -30,6 +30,12 @@ defaults:
     etcd:
       kvName: arohcp-etcd-{{ .ctx.regionShort }}
       kvSoftDelete: true
+    istio:
+      url: "https://istio.io/downloadIstio"
+      istioctlVersion: "1.23.1"
+      tag: "prod-stable"
+      version: "asm-1-22"
+      newVersion: ""
```

Comment:
A quick overview of why we need this kind of value would be useful. As I remember it there's some namespace selection going on, but it would be good to confirm.

I don't know how this value gets corrupted when we have concurrent upgrades going on. If we had an upgrade from 1 to 2 running but failed, then we tried to upgrade to 3, we'd need to have config that specified 1->3, since 2->3 wouldn't pick up regions that still were on 1? I don't know if we want that.

Reply:
Actually, I don't see the old version used for anything in the script - https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/hcp?path=/ev2/svc/istio/archive/istio.sh&version=GBmain&line=13&lineEnd=14&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Are we sure we need both parameters?

Reply:
so like i wrote it in the [Readme](https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/hcp?path=/ev2/svc/istio/README.md&_a=preview&anchor=istio-upgrade-procedue.)
So basically 
1. we need to add istio new version and old version in to SVC infra pipeline. and run it( so svc cluster would have both version in the cluster)
2. Run the istio pipeline to replace the version.
3. Update the svc config to have the new version, and run the svc infra again. 

Sorry for the confusion on having the old version in the script. it was a legacy code I forgot to delete from previous version of the shell script. I will delete it.